meetings/2017-12-06.md

```diff
@@ -0,0 +1,163 @@
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2017-12-06
+
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=1sHufGWR_WU
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/432
+* **Minutes Google Doc**: https://docs.google.com/document/d/1N029ttRxw92pv1k0Xc0-abkwbHMRW7Ts6r4nFeJbF4M
+
+## Present
+
+* Colin Ihrig @cjihrig (TSC)
+* Franziska Hinkelmann @fhinkel (TSC)
+* James M Snell @jasnell (TSC)
+* Jeremiah Senkpiel @Fishrock123 (TSC)
+* Matteo Collina @mcollina (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Ali Sheikh @ofrobots (TSC)
+* Rod Vagg @rvagg (TSC)
+* Trevor Norris @trevnorris (TSC)
+* Rich Trott @Trott (TSC)
+* Daniel Bevenius @danbev (observer)
+* William Kapke @williamkapke (observer)
+
+## Agenda
+
+### Announcements
+
+* No announcements this week.
+ 
+*Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* Enable explicit `.m.js` intent for ESM [#16170](https://github.com/nodejs/node/pull/16170)
+  * asked to provide clear statement on what is negotiable and what is not
+  * Myles has been working on blog post to cover that, going out next Tuesday
+  * Myles had talked to originator of issue, discussed content, seemed easy.
+  * can remove from agenda
+
+* buffer: runtime-deprecate Buffer(num) by default [#15608](https://github.com/nodejs/node/pull/15608)
+  * phased approach to deprecate some parts earlier
+  * question to TSC is that an option ?
+  * opinion from TSC in issue has mostly been -> not an option
+  * no opposition from those in meeting, will remove from agenda and say 'not an option' in 
+    issue.
+
+* events: add off alias to removeListener [#17156](https://github.com/nodejs/node/pull/17156)
+  * Matteo mentioned he was opposed
+  * Rod not a fan, but people seem to really want it
+  * Opposed to be it being semver minor.  Believe it is a bad idea but more ok if it is
+    semver minor.  
+  * Trevor ran into issues in past when we added an _property.  Cannot add to event
```

Comment:
needs end `_` to close the italics

Reply:
I was meant to be underscore property, I'll just put that into words